cea/optimization/master/cost_model.py

```diff
@@ -125,8 +125,9 @@ def addCosts(indCombi, buildList, locator, dicoSupply, QUncoveredDesign, QUncove
         # Furnace
         if dicoSupply.Furnace_on == 1:
             P_design = dicoSupply.Furnace_Q_max
-            
-            fNameSlavePP = dicoSupply.configKey + "PPActivationPattern.csv"
+
+            fNameSlavePP = locator.optimization_slave_PPActivationPattern(dicoSupply.configKey)
+            # fNameSlavePP = dicoSupply.configKey + "PPActivationPattern.csv"
```

Comment:
delete lines you're replacing. don't comment them out! this just creates a mess nobody will ever be able to understand! (we use git to track what the line used to be)